setup.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
```

Comment:
Could this file be located in src/python/tempusloader rather than at the root ?
We may have other packages for the QGIS plugin and/or python libs ...